.github/workflows/mlir-tensorrt-ci.yml

```diff
@@ -235,7 +235,7 @@ jobs:
               -DCMAKE_BUILD_TYPE=RelWithDebInfo \
               -DMLIR_TRT_PACKAGE_CACHE_DIR=/.cache.cpm \
               -DMLIR_TRT_ENABLE_ASSERTIONS=ON \
-              -DMLIR_TRT_DOWNLOAD_TENSORRT_VERSION=9.2.0.5 \
+              -DMLIR_TRT_DOWNLOAD_TENSORRT_VERSION=8.6.1.6 \
```

Comment:
Looks like this won't work

Reply:
`cmake --preset ninja-llvm -DMLIR_TRT_DOWNLOAD_TENSORRT_VERSION=8.6.1.6`
does download TRT 8.6.1.6.